src/index.js

```diff
@@ -31,6 +31,7 @@ const plugin = {
             if (directives.hasOwnProperty(prop)) {
                 const directive = directives[prop]
                 Vue.directive(directive.name, directive)
+                console.log(directive.name)
```

Comment:
```suggestion
```

Reply:
re: overlapping, yes, very aware, although struggled to amend it. Have given the tooltip a `z-index: 1`, and ensured all parents are equitable, but it still wasn't playing ball.